streamer/src/streamer.rs

```diff
@@ -361,6 +361,10 @@ impl StakedNodes {
             .filter(|&stake| stake > 0);
         let total_stake = values.clone().sum();
         let (min_stake, max_stake) = values.minmax().into_option().unwrap_or_default();
+        info!(
```

Comment:
it seems more reasonable to put this in the calling code - you don't need to log this for the calculation.

Reply:
will fix